cookbook/deployment/heroku.rst

```diff
@@ -305,20 +305,20 @@ This is also very useful to build assets on the production system, e.g. with Ass
 
     With the next deploy, Heroku compiles your app using the Node.js buildpack and
     your npm packages become installed. On the other hand, your ``composer.json`` is
-    now ignored. To compile your app with both buildpacks, Node.js *and* PHP, you can
-    use a special `multiple buildpack`_. To override buildpack auto-detection, you
```

Comment:
the ``multiple buildpack`` link should be removed at the bottom of the page too, as you don't use it anymore